bin/templates/scripts/cordova/lib/versions.js

```diff
@@ -49,9 +49,9 @@ exports.get_apple_xcode_version = () => {
     return spawn('xcodebuild', ['-version'])
         .then(output => {
             const versionMatch = /Xcode (.*)/.exec(output);
-
-            if (!versionMatch) return Promise.reject(output);
-
+            if (!versionMatch) {
+                throw new CordovaError('Could not determine Xcode version');
```

Comment:
I am OK with the simplified message. Could the `output` contain additional information that would help users debug why it failed?

If it could, maybe only include the output with the `--verbose` flag?

But I also don't know if the flag is visible here.

Anyways, I am good with these changes as is.

Reply:
Good Point! I included the output in the error message.